Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/fallthrough/FallThroughDialog.java

```diff
@@ -66,23 +66,23 @@
 	}
 
 	/**
-	 * @see ghidra.util.bean.GhidraDialog#applyCallback()
+	 * @see DialogComponentProvider#applyCallback()
```

Comment:
removable

I'm really just marking them now so they don't fall through the cracks.

Reply:
See https://github.com/NationalSecurityAgency/ghidra/pull/1755#discussion_r407329830